catalyst-gateway/bin/src/service/api/mod.rs

```diff
@@ -23,6 +27,9 @@ const API_TITLE: &str = "Catalyst Gateway";
 /// The version of the API
 const API_VERSION: &str = "1.2.0";
 
+/// Port
+const PORT: &str = "5432";
```

Comment:
Combine the service settings and doc settings.
Extract the port from the server socket address, rather than hardcoding it.